### PR TITLE
SAMSON-217 fix brakeman warnings

### PR DESCRIPTION
**Pull request by grosser**
https://zendesk.atlassian.net/browse/SAMSON-217

ignored warning tracked in https://support.zendesk.com/agent/tickets/1542041

### Risks
 - None

### CODE REVIEW

**Comment by grosser** [comment]
@zendesk/samson

File: app/controllers/stages_controller.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@ def destroy
   end
 
   def reorder
-    Stage.reorder(params[:stage_id])
```
Comment (grosser) [inline app/controllers/stages_controller.rb]: was overriding a activerecord method and scaring brakeman

**Comment by jonmoter** [comment]
@grosser, I don't think Travis is actually running Brakeman.  When I look at the [Travis build](https://travis-ci.org/zendesk/samson/jobs/119936571) it says this:

```
$ script/ci
Unknown task

The command "script/ci" exited with 0.
```

**Comment by jonmoter** [comment]
And script/ci should probably exit with a non-zero value if you try running an unknown task.

**Comment by grosser** [comment]
all green .. got rid of script/ci too and made `rake test:js` run standalone :)

**Comment by jonmoter** [comment]
:+1:

**Pull request merged**